src/views/monitoring/alert-manager/pages/AlertDetailPage.vue

```diff
@@ -179,15 +162,21 @@ export default {
 
         const alertTitleEditConfirm = async () => {
             state.alertTitleEditFormVisible = false;
-            await getAlertData();
         };
 
         (async () => {
-            await Promise.all([
-                store.dispatch('resource/webhook/load'),
-                store.dispatch('resource/user/load'),
-                getAlertData(),
-            ]);
+            state.loading = true;
+            try {
+                await Promise.allSettled([
+                    store.dispatch('resource/webhook/load'),
+                    store.dispatch('resource/user/load'),
+                    await store.dispatch('service/alert/getAlertData', props.id),
```

Comment:
엇 여기에 await 이 들어갔어요!
```suggestion
                    store.dispatch('service/alert/getAlertData', props.id),
```